src/wallet/wallet.cpp

```diff
@@ -2197,7 +2197,7 @@ CAmount CWallet::GetAvailableBalance(const CCoinControl* coinControl) const
 
     CAmount balance = 0;
     std::vector<COutput> vCoins;
-    AvailableCoins(vCoins, true, coinControl);
+    AvailableCoins(vCoins, coinControl);
```

Comment:
Not an actual refactor because this changes behavior. But in practice AMT there's no call to `GetAvailableBalance` with `m_include_unsafe_inputs = true`.

Reply:
Good catch, it does change behavior here but is backwards-compatible so I think it should be ok.
If later we want to make `GetAvailableBalance` with `m_include_unsafe_inputs = true` possible, we can use `coinControl` for that.